.wordlist.txt

```diff
@@ -100,3 +100,18 @@ Tanzu
 OpenShift
 RHCOS
 RHEL
+SSM
+OwnerId
+aws
+GovCloud
+minimized
+Anbox
+ImageLocation
+VIRT
+hvm
+pv
+ebs
+gp
+io
+xenial
+discoverable
```

Comment:
won't be needed if the change suggested below is used.

Reply:
Done